{"name": "test_login[invalid_user-invalid_pass-False]", "status": "failed", "statusDetails": {"message": "AssertionError: 未检测到错误弹窗\nassert None is not None", "trace": "self = <flutter_app_test.testcases.test_login.TestLogin object at 0x000002305B4DEE90>, username = 'invalid_user'\npassword = 'invalid_pass', expected = False\n\n    @pytest.mark.parametrize(\"username,password,expected\", [\n        (\"invalid_user\", \"invalid_pass\", False),\n        (\"2072583546\", \"np123456\", True),\n    ])\n    def test_login(self, username, password, expected):\n        login_page = LoginPage(self.driver)\n        login_page.login(username, password)\n        if expected:\n            # 登录成功断言\n            assert login_page.is_home_button_present(), \"登录后未检测到首页按钮，登录失败\"\n        else:\n            # 登录失败断言\n            error_msg = login_page.get_error_message()\n>           assert error_msg is not None, \"未检测到错误弹窗\"\nE           AssertionError: 未检测到错误弹窗\nE           assert None is not None\n\ntestcases\\test_login.py:55: AssertionError"}, "parameters": [{"name": "username", "value": "'invalid_user'"}, {"name": "password", "value": "'invalid_pass'"}, {"name": "expected", "value": "False"}], "start": 1748515290926, "stop": 1748515309965, "uuid": "6d09011d-c917-4da4-baff-32a33ccdc982", "historyId": "1b6026c39fcb6726426cb1cdc25e9381", "testCaseId": "93a39feb8eb701231e6c069b76eeac98", "fullName": "testcases.test_login.TestLogin#test_login", "labels": [{"name": "feature", "value": "登录功能"}, {"name": "parentSuite", "value": "testcases"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-QVK4EQ1"}, {"name": "thread", "value": "33768-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.test_login"}]}